src/admin-views/rsvp-metabox-not-going.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Admin setting for "Can't Go" checkbox when creating/editing RSVP in Classic Editor.
+ */
+?>
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: adaab674d61193fb55e069f0a4c5e78f</sub>